### PR TITLE
announce foundation

### PR DESCRIPTION
**Pull request by maxandersen**

### CODE REVIEW

File: _posts/2024-07-31-quarkus-moving-to-commonhaus.adoc [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+## Why Move to a Foundation?
+
+Quarkus has seen tremendous growth and adoption, especially with the recent advancements in Java and Kubernetes-native development. Despite our successes, our vibrant and ever-expanding Quarkus community with over 900 contributors, there is still the perception that Quarkus is too dependent on Red Hat. We aim to change this by providing a neutral ground where other organisations and contributors can feel equally valued and involved.
+
+## Why Commonhaus?
+
+Commonhaus stands out with its innovative governance framework and commitment to independently managed projects. Joining Commonhaus allows Quarkus to benefit from a self-governing model. Commonhaus suits very well the fast delivery model of Quarkus. We are excited to join the other ten prominent projects that are hosted in Commonhaus, which include Hibernate and Jackson, prominent components of Quarkus.
```
Comment (holly-cummins) [inline _posts/2024-07-31-quarkus-moving-to-commonhaus.adoc:22]:
```suggestion
Commonhaus stands out with its innovative governance framework and commitment to independently managed projects. Joining Commonhaus allows Quarkus to benefit from a self-governing model. Commonhaus suits very well the fast delivery model of Quarkus. We are excited to join the other ten prominent projects that are hosted in Commonhaus, which include Hibernate and Jackson, important components of Quarkus.
```

Best not to repeat 'prominent'.

File: _posts/2024-07-31-quarkus-moving-to-commonhaus.adoc [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+## What's Next?
+
+Quarkus will continue to innovate and evolve. We are dedicated to making Quarkus the best framework for Java development. This transition will enable us to welcome more contributions from a diverse range of developers and organisations. We are actively working on upcoming releases and are eager to hear your ideas and feedback.
+
+This move is as much for Quarkus as it is for you. If you or the organisation you work at are interested in ensuring that the Quarkus ecosystem grows and evolves in a direction that helps you, come and join us in helping Quarkus transition to the CommonHaus foundation. 
+
+CommonHaus is currently bootstrapping, enabling it to be shaped and moulded to help you and opensource projects like Quarkus to have a great home.
```
Comment (holly-cummins) [inline _posts/2024-07-31-quarkus-moving-to-commonhaus.adoc:30]:
```suggestion
Commonhaus is currently bootstrapping, enabling it to be shaped and moulded to help you and opensource projects like Quarkus to have a great home. 
```

File: foundation-faq.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
+---
+layout: usage
+title: Quarkus at CommonHaus FAQ
```
Comment (holly-cummins) [inline foundation-faq.md:3]:
```suggestion
title: Quarkus at Commonhaus FAQ
```

File: foundation-faq.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+See Commonhaus’ own [FAQ](https://www.commonhaus.org/about/faq.html).
+
+## What does the move consist of exactly?
+
+The move consists of donating intellectual property owned by Red Hat and related to Quarkus projects.
+The donated intellectual property includes in particular:
+
+ * the Quarkus trademark;
+ * related domain names.
```
Comment (holly-cummins) [inline foundation-faq.md:36]:
```suggestion
 * the Quarkus trademark
 * related domain names
```

File: _posts/2024-07-31-quarkus-moving-to-commonhaus.adoc [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+Commonhaus stands out with its innovative governance framework and commitment to independently managed projects. Joining Commonhaus allows Quarkus to benefit from a self-governing model. Commonhaus suits very well the fast delivery model of Quarkus. We are excited to join the other ten prominent projects that are hosted in Commonhaus, which include Hibernate and Jackson, important components of Quarkus.
+
+## What's Next?
+
+Quarkus will continue to innovate and evolve. We are dedicated to making Quarkus the best framework for Java development. This transition will enable us to welcome more contributions from a diverse range of developers and organisations. We are actively working on upcoming releases and are eager to hear your ideas and feedback.
+
+This move is as much for Quarkus as it is for you. If you or the organisation you work at are interested in ensuring that the Quarkus ecosystem grows and evolves in a direction that helps you, come and join us in helping Quarkus transition to the CommonHaus foundation. 
+
+Commonhaus is currently bootstrapping, enabling it to be shaped and moulded to help you and opensource projects like Quarkus to have a great home.
```
Comment (gsmet) [inline _posts/2024-07-31-quarkus-moving-to-commonhaus.adoc:30]:
```suggestion
Commonhaus is currently bootstrapping, enabling it to be shaped and moulded to help you and open source projects like Quarkus to have a great home. 
```

File: foundation-faq.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+The main reason for choosing Commonhaus over another foundation was its governance framework, which emphasises project self-governance.
+See, in particular, its guiding principles.
+
+## Why not [put your favourite foundation here]?
+
+See Commonhaus’ own [FAQ](https://www.commonhaus.org/about/faq.html).
+
+## What does the move consist of exactly?
+
+The move consists of donating intellectual property owned by Red Hat and related to Quarkus projects.
```
Comment (holly-cummins) [inline foundation-faq.md:32]:
```suggestion
    The move consists of donating Quarkus-related intellectual property owned by Red Hat.
```

File: foundation-faq.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+Red Hat’s involvement in Quarkus remains as strong as ever.
+
+The Quarkus team is expecting increased collaboration between Red Hat and other companies, which may require setting time aside to get new contributors on board.
+
+## What is changing in practice for contributors to Quarkus projects?
+
+Commonhaus requires Quarkus to adhere to its [policies](https://www.commonhaus.org/policies/), including in particular its [intellectual property policy]https://www.commonhaus.org/policies/ip-policy/) and [trademark policy](https://www.commonhaus.org/policies/trademark-policy/).
+
+This is great news for you all, as it legally binds the project to remain open-source forever.
```
Comment (gsmet) [inline foundation-faq.md:54]:
```suggestion
This is great news for you all, as it legally binds the project to remain open source forever.
```

File: foundation.md [hunk 8]
```diff
@@ -0,0 +1,14 @@
+---
+layout: usage
+title: Quarkus at Foundation
+subtitle: Help us make Quarkus even better, in a vendor neutral way.
+permalink: /foundation/
+---
+
+Quarkus is in the process of joining the https://commonhaus.org[CommonHaus Foundation], a vendor-neutral foundation that will provide a home for the Quarkus project. The foundation will be a place where the community can collaborate on the future of Quarkus, and where the project can grow and evolve in a vendor-neutral way.
```
Comment (gsmet) [inline foundation.md:8]:
```suggestion
Quarkus is in the process of joining the https://commonhaus.org[Commonhaus Foundation], a vendor-neutral foundation that will provide a home for the Quarkus project. The foundation will be a place where the community can collaborate on the future of Quarkus, and where the project can grow and evolve in a vendor-neutral way.
```

File: foundation.md [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+
+For more information see:
+
+* The intent to join a foundation [blog](/blog/quarkus-at-foundation/)
+* The announcement on joining the CommonHaus Foundation [blog](/blog/quarkus-moving-to-commonhaus/)
```
Comment (gsmet) [inline foundation.md:13]:
```suggestion
* The announcement on joining the Commonhaus Foundation [blog](/blog/quarkus-moving-to-commonhaus/)
```

**Review by gsmet — CHANGES_REQUESTED** [review]
LGTM. I suggested a few micro changes.

File: foundation-faq.md [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+
+Commonhaus also requires having a code of conduct, and provides a recommended one, but each project has flexibility on such details so we could decide to customise it.
+
+Quarkus contributors can opt to become Commonhaus Members; this is not a requirement to participate in the Quarkus project but is a requirement to help define Commonhaus.
```
Comment (holly-cummins) [inline foundation-faq.md:58]:
```suggestion
Quarkus contributors can opt to become [Commonhaus Members](https://www.commonhaus.org/community/#become-a-member); membership is not a requirement to participate in the Quarkus project, but members can also help shape the future of Commonhaus itself.
```

Not totally sure what this sentence is saying. Is it saying that if people want to help shape the future direction of Commonhaus they need to become members? I'll assume so.

File: foundation.md (hunk 8 above)
Comment (holly-cummins) [inline foundation.md]:
```suggestion
Quarkus is in the process of joining the https://commonhaus.org[Commonhaus Foundation], a vendor-neutral foundation that will provide a home for the Quarkus project. The foundation will be a place where the community can collaborate on the future of Quarkus, and where the project can grow and evolve in a vendor-neutral way.
```

File: foundation.md (hunk 9 above)
Comment (holly-cummins) [inline foundation.md]:
```suggestion
* The announcement on joining the Commonhaus Foundation [blog](/blog/quarkus-moving-to-commonhaus/)
```

File: _posts/2024-07-31-quarkus-moving-to-commonhaus.adoc [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+
+## Why Commonhaus?
+
+Commonhaus stands out with its innovative governance framework and commitment to independently managed projects. Joining Commonhaus allows Quarkus to benefit from a self-governing model. Commonhaus suits very well the fast delivery model of Quarkus. We are excited to join the other ten prominent projects that are hosted in Commonhaus, which include Hibernate and Jackson, prominent components of Quarkus.
+
+## What's Next?
+
+Quarkus will continue to innovate and evolve. We are dedicated to making Quarkus the best framework for Java development. This transition will enable us to welcome more contributions from a diverse range of developers and organisations. We are actively working on upcoming releases and are eager to hear your ideas and feedback.
+
+This move is as much for Quarkus as it is for you. If you or the organisation you work at are interested in ensuring that the Quarkus ecosystem grows and evolves in a direction that helps you, come and join us in helping Quarkus transition to the CommonHaus foundation.
```
Comment (holly-cummins) [inline _posts/2024-07-31-quarkus-moving-to-commonhaus.adoc:28]:
```suggestion
This move is as much for Quarkus as it is for you. If you or the organisation you work at are interested in ensuring that the Quarkus ecosystem grows and evolves in a direction that helps you, come and join us in helping Quarkus transition to the Commonhaus foundation. 
```

File: foundation-faq.md (hunk 10 above)
Comment (maxandersen) [reply foundation-faq.md:58]: yeah - missed this one before pushing but I'll adjust it in follow up commit

File: foundation-faq.md [hunk 12]
```diff
@@ -0,0 +1,60 @@
+---
+layout: usage
+title: Quarkus at Commonhaus FAQ
+subtitle: Frequently Asked Questions
+permalink: /foundation/faq
+---
+
+## Why move to a foundation?
+
+Enable us to welcome more contributions from a diverse range of developers and organisations.
```
Comment (holly-cummins) [inline foundation-faq.md:10]: The connection here is a bit weak, to me. Are we trying to say that at the moment we're *prevented* from welcoming contributions (I don't think that's true, but it's kind of implied by the phrasing), or are we saying that we hope having a home outside Red Hat will make contributing to Quarkus more attractive to more people. Or are we saying that if the governance is formally independent of Red Hat more people can become involved in the governance/really get involved in core Quarkus? Or that we want to correct a perception that people outside Red Hat can't contribute? I think we probably want to be saying more of the latter three, and less of the first.

**Pull request merged**